docs/streams/developer-guide/processor-api.html

```diff
@@ -396,6 +397,21 @@ <h2>
                 </ul>
                 </p>
             </div>
+            <div class="section" id="readonly-state-stores">
+                <span id="streams-developer-guide-state-store-readonly"></span><h3><a class="toc-backref" href="#id12">ReadOnly State Stores</a><a class="headerlink" href="#readonly-state-stores" title="Permalink to this headline"></a></h3>
+                <p>The changelog topic for a fault-tolerant state store can also be used to perform lookups against. It is
```

Comment:
This sentence is somewhat confusing. Maybe we should start with something like this:

> A read-only state store materialized the data from its input topic. It also uses the input topic for fault-tolerance, and thus does not have an additional changelog topic (the input topic is re-used as changelog). Thus, the input topic should be configured with log compaction (add link to log compaction). Note that no other processor should modify the content of the state store, and the only writer should be the associated "state update processor"; other processor may read the content of the read-only store.

Reply:
a lot better indeed. Modified it.